client/my-sites/stats/all-time-highlights-section/index.tsx

```diff
@@ -151,13 +154,25 @@ export default function AllTimeHighlightsSection( {
 		};
 	}, [ isStatsLoading, translate, views, viewsBestDay, viewsBestDayTotal, userLocale ] );
 
+	// TODO: Investigate granularity on Insights page.
+	// Do we want to get granular on this page? It's probably not necessary yet.
+	const shouldLockCards = useShouldGateStats( STAT_TYPE_INIGHTS );
```

Comment:
Currently testing at the page level rather than the feature level. ie: These cards should be locked if the Insights page is locked. I'm not sure we need to get more granular unless we decide to expose different widgets based on plans/tiers.

Reply:
I think it would be better to get a feature for each card as we would like to know which card drives more sales than the others.

Reply:
That would be a separate issue though, no?

My thinking is that we don't need granularity when checking to gate stats because it's on/off at this point. We could still send different tracks events for the individual button clicks to understand what drives sales though. I guess the constants could be defined and used for the tracks events but aren't necessary for widget/feature gating.

Reply:
I think it would make sense to use `STAT_TYPE_INIGHTS_ALL_TIME_HIGHLIGHTS` to paywall the three cards since they seem to be relevant as put together but not independent like modules.